sunpy/net/hek/attrs.py

```diff
@@ -590,16 +585,3 @@ class Misc:
     PeakPower = _StringParamAttrWrapper('PeakPower')
     PeakPowerUnit = _StringParamAttrWrapper('PeakPowerUnit')
     RasterScanType = _StringParamAttrWrapper('RasterScanType')
-
-
-# Deprecate old classes
```

Comment:
Do we still not need this?

Reply:
I moved it to just a Time class higher up the file.